community/icinga2/APKBUILD

```diff
@@ -14,26 +14,20 @@ install="$pkgname.pre-install $pkgname.post-install"
 pkgusers="icinga"
 pkggroups="icinga icingacmd"
 subpackages="$pkgname-doc $pkgname-bash-completion:bashcomp:noarch $pkgname-vim::noarch $pkgname-openrc"
-source="$pkgname-$pkgver.tar.gz::https://github.com/Icinga/$pkgname/archive/v$pkgver.tar.gz
+source="icinga2-$pkgver.tar.gz::https://github.com/Icinga/icinga2/archive/v$pkgver.tar.gz
```

Comment:
Nitpick, but I think we usually keep the `$pkgname` in the archive name, since it makes sense to have the name of the package in the distfile since that makes mapping the distfile to the respective package easier.